shared/src/commonMain/kotlin/com/baijum/ukufretboard/data/sync/BackupCodec.kt

```
@@ -0,0 +1,432 @@
+                } else {
+                    add(elem)
+                }
+            }
+        }
+    }
+
+    private val iosFingerToKMP = mapOf(
+        "T" to "THUMB", "I" to "INDEX", "M" to "MIDDLE", "R" to "RING", "P" to "PINKY",
+    )
+
+    private fun normalizeIosFingerpicking(root: JsonObject): JsonArray? {
```

detekt (1.23.8)

[warning] The function normalizeIosFingerpicking appears to be too complex based on Cognitive Complexity (complexity: 40). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)